app/views/components/_related-navigation.html.erb

```diff
@@ -19,7 +19,7 @@
     <h2 id="related-nav-related_items" class="app-c-related-navigation__main-heading" >
       <%= t('components.related_navigation.related_content') %>
     </h2>
-    <% related_content.each do |section| %>
+    <% related_content.each_with_index do |section, section_index| %>
```

Comment:
This could be `related_content.each.with_index(1) do |section, section_index|` which would start the index at 1 and simplify line 47. It's also consistent with how the links are iterated below.

Reply:
@steventux Had to change the logic slightly anyway as the section_index was being incremented even for sections without any links to display.